VAMobile/src/testUtils.tsx

```diff
@@ -231,8 +231,19 @@ ctxFn.skip = (name: string, fn: () => void) => {
 
 export const context = ctxFn
 
+export type QueriesData = Array<{
+  queryKey: QueryKey
+  data: any
+}>
+
+type RenderParams = {
+  preloadedState?: any // TODO: Update this type to Partial<RootState> and fix broken tests
```

Comment:
Some of the tests are broken when the type is set for this because of incorrectly formatted state. This parameter may not even be needed once we complete the React Query migration, but I left a comment here just in case